portfolio/src/main/webapp/script.js

```diff
@@ -79,11 +79,11 @@ function addRandomFact(){
         facts.splice(idx,1);
         // Add it to the page.
         factContainer.innerText = fact;
-    }
+        }
 }
 
 function getMessages(){
-    fetch('/data').then(response => response.json()).then((comments) => {
+    fetch('/comments').then(response => response.json()).then((comments) => {
```

Comment:
It sounds like you rename "/data" to "/messages" in the Java file. Should you also do the same renaming here?

Reply:
Ack